packages/site/src/components/Analytics.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router";
+import { useOnMount } from "@jobber/hooks";
+import ReactGA from "react-ga4";
+
+const IS_LOCAL = window.location.hostname.includes("localhost");
+
+/**
+ * Analytics tracks page navigations and reports them to our analytics provider.
+ */
+export function Analytics() {
```

Comment:
For a V0.1 this is perfect. Very easily extendable but capturing all page transitions using the router, perfect!